pkg/apis/autoscaling/types.go

```diff
@@ -157,10 +162,19 @@ type HPAScalingRules struct {
 	// If not set, the default value MaxPolicySelect is used.
 	// +optional
 	SelectPolicy *ScalingPolicySelect
-	// policies is a list of potential scaling polices which can used during scaling.
-	// At least one policy must be specified, otherwise the HPAScalingRules will be discarded as invalid
+	// policies is a list of potential scaling polices which can be used during scaling.
+	// If not set, use the default values:
+	// - For scale up: allow doubling the number of pods, and an absolute change of 4 pods in a 15s window.
```

Comment:
```suggestion
	// - For scale up: allow doubling the number of pods, or an absolute change of 4 pods in a 15s window.
```

Should this be an `or`, since the SelectPolicy will select one eitehr the max or min of these